### ISSUE TITLE
package.json in 2.0.0

### ISSUE THREAD

**Issue by jakub-g** [dailymotion/vast-client-js#236]
In 2.0.0 `main` field in `package.json` points to `src/index.js`
https://github.com/dailymotion/vast-client-js/blob/master/package.json#L25
...which is an ES module
https://github.com/dailymotion/vast-client-js/blob/master/src/index.js
...which means AFAIU that node versions that do not support ES modules (seems like no version of node supports it now, except 10.x behind the flag?) will fail to load it.
(in 1.x `main` was pointing to `dist/index.js`).

AFAIU `main` should point to a ES5 output, whereas a new field `module` could point to the ES6 code
https://webpack.js.org/guides/author-libraries/#final-steps

Also `"browser": "vast-client.js"` should be added as it was done in 1.x branch, otherwise problems described in #156 will come back

**Comment by eliamaino-fp** [comment]
I agree, so you suggest to have something like this?
```javascript
...
"main" : "vast-client.js",
"module" : "src/index.js",
"browser" : "vast-client.js"
...
```

**Comment by jakub-g** [comment]
I think this should be fine:
```...
"module" : "src/index.js",
"browser" : "vast-client.js"
...```
but this
```
"main" : "vast-client.js",
```
...I guess it wouldn't work in node because it's just building the lib and exporting as a variable (check the generated `vast-client.js`) , and we want `module.exports`.

Probably we will need two separate webpack configs, for browser output and node output. I'm not that familiar how this stuff is handled typically, to be researched. Anyway, AFAIU, we'll probably end up with 3 different entry points:

```
"main" : "vast-client-node.js",
"module" : "src/index.js",
"browser" : "vast-client.js"
```

**Comment by eliamaino-fp** [comment]
Fixed in https://github.com/dailymotion/vast-client-js/pull/237
And released in [`v2.0.1`](https://github.com/dailymotion/vast-client-js/releases/tag/2.0.1)
